pkg/VSSetup/VSSetup.signproj

```diff
@@ -1,21 +1,20 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Sign" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\..\packages\MicroBuild.Core.0.3.0\build\MicroBuild.Core.props" Condition="Exists('..\..\packages\MicroBuild.Core.0.3.0\build\MicroBuild.Core.props')" />
+<Project ToolsVersion="Current" Sdk="Microsoft.Build.NoTargets/3.7.56">
   <PropertyGroup>
-    <Configuration Condition="'$(Configuration)'==''">Debug</Configuration>
     <SolutionDir Condition="'$(SolutionDir)'==''">..\..\</SolutionDir>
     <SolutionDir Condition="!HasTrailingSlash('$(SolutionDir)')">$(SolutionDir)\</SolutionDir>
     <OutDir Condition="'$(OutDir)'==''">$(SolutionDir)bin\$(Configuration)\</OutDir>
     <OutDir Condition="!HasTrailingSlash('$(OutDir)')">$(OutDir)\</OutDir>
     <IntermediateOutputPath Condition="'$(IntermediateOutputPath)'==''">$(SolutionDir)obj\$(Configuration)\</IntermediateOutputPath>
     <IntermediateOutputPath Condition="!HasTrailingSlash('$(IntermediateOutputPath)')">$(IntermediateOutputPath)\</IntermediateOutputPath>
-    <SignDependsOn>
-        PrepareForBuild;
-        $(SignDependsOn);
-    </SignDependsOn>
+    <TargetFramework>net48</TargetFramework>
```

Comment:
Happy to make that change, but does it matter for the signproj? The PS module project is still using net35 so that should still capture what we need. Should we update off of net35 at this point?

Reply:
Oh, sorry. I missed that it was the `.signproj`. Never mind.

Yeah, may as well update the PS module project to `net462`.